playbooks/install_stack.yaml

```
@@ -205,11 +205,19 @@
           octavia_env:
             - /usr/share/openstack-tripleo-heat-templates/environments/services/octavia.yaml
 
+  - name: Generate container_image_prepare.yaml if not using rhos-release
+    when:
+      - cip_config is not defined
+      - ansible_facts.distribution == 'RedHat'
+      - rhsm_enabled
+    command: openstack tripleo container image prepare default --output-env-file /home/stack/containers-prepare-parameters.yaml
```

I've seen `/home/stack` being hardcoded in a few places and `{{ ansible_env.HOME }}` at other places. I think we should prefer `{{ ansible_env.HOME }}` for consistency:
```suggestion
    command: openstack tripleo container image prepare default --output-env-file {{ ansible_env.HOME }}/containers-prepare-parameters.yaml
```

No need to make the change here, we should make the change globally in a separate PR.

---

I'll do it.